app/components/publications/publications-filter-modal.js

```diff
@@ -0,0 +1,40 @@
+import Component from '@glimmer/component';
+import {
+  action, set
+} from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class PublicationsFilter extends Component {
+  @tracked filter;
+
+  constructor() {
+    super(...arguments);
+    this.filter = this.args.filter;
+  }
+
+  initialize() {
+    this.filter = this.args.filter;
+  }
+
+  @action
+  cancel() {
+    this.args.onCancel();
+  }
+
+  @action
+  save() {
+    this.args.onSave(this.filter);
+  }
+
+  @action
+  reset() {
+    this.args.onReset();
```

Comment:
Gezien deze component enkel voor dit specifieke doel (publicatie-filter) gebruikt wordt, zou ik ervoor durven argumenteren om het resetten van de state intern aan de component te houden, maar da's niet zoals de story het definiëert zie ik nu. My 2 cents, maar zeker geen must.

Reply:
Reset gebeurt in de component nu